src/main/resources/sql/bridge_healthdatarecord3.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS bridge_healthdatarecord3_<dateSuffix> (
+    id              CHAR(36)        ENCODE RAW          NOT NULL,
+    healthCode      CHAR(36)        ENCODE RAW          NOT NULL,
+    createdOn       BIGINT          ENCODE DELTA32k             ,
+    studyId         VARCHAR(50)     ENCODE BYTEDICT             ,
+    uploadId        CHAR(36)        ENCODE RAW          NOT NULL,
+    uploadDate      DATE            ENCODE RAW                  ,
+    schemaId        VARCHAR(250)    ENCODE LZO                  ,
```

Comment:
There are ~100 schemaIds, and this set grows very slowly. Would it be worthwhile to use a bytedict instead?